### ISSUE TITLE
module 'tensorflow.keras.preprocessing' has no attribute 'image_dataset_from_directory'

### ISSUE THREAD

**Issue by github4mathews** [SciSharp/SciSharp-Stack-Examples#23]
module 'tensorflow.keras.preprocessing' has no attribute 'image_dataset_from_directory'

train_ds = tf.keras.preprocessing.image_dataset_from_directory(
  data_dir,
  validation_split=0.2,
  subset="training",
  seed=123,
  image_size=(img_height, img_width),
  batch_size=batch_size)

in image_classification.py

**Comment by Oceania2018** [comment]
There it is but with bug for this moment.

![image](https://user-images.githubusercontent.com/1705364/97082596-81344780-15d0-11eb-84da-d18d9ceaf525.png)

**Comment by pardont** [comment]
@Oceania2018 is the bug fixed?  NullReferenceException thrown when executing the code .
https://github.com/SciSharp/SciSharp-Stack-Examples/issues/27

**Comment by Oceania2018** [comment]
@pardont Not yet.

**Comment by lqdev** [comment]
@Oceania2018 is there a recommended workaround for this in the meantime?

**Comment by Oceania2018** [comment]
@lqdev Same issue as https://github.com/SciSharp/TensorFlow.NET/issues/666, fixed the `image_dataset_from_directory`, but still has other bug for the rest part of that example.
